src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/1.sql

```diff
@@ -0,0 +1,8 @@
+-- Enable RCSI
+ALTER DATABASE CURRENT SET READ_COMMITTED_SNAPSHOT ON;
+
+-- Create Resource table
+IF NOT EXISTS ( SELECT * FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_NAME = 'Resources' )
```

Comment:
I was thinking that the initial script could be destructive. Drop everything and re-create. Not what we would do for later versions of course :)

Reply:
That's an interesting thought and possibly dangerous... I think it merits some deeper thought around the workflows for development and what actually happens when deployed.

Reply:
Punting for further discussion.